plankton/devices/linkam_t95/interfaces/stream_interface.py

```diff
@@ -21,6 +21,19 @@
 
 
 class LinkamT95StreamInterface(StreamAdapter):
+    """
+    Linkam T95 TCP stream interface
+
+    This is the interface of a simulated Linkam T95 device. The device listens on a configured
+    host:port-combination, one option to connect to it is via telnet:
+
+        $ telnet host port
+
+    Once connected, it's possible to send the specified commands, described in the dynamically
+    generated documentation. Information about host, port and line terminators in the concrete
+    device instance are also generated dynamically.
```

Comment:
As we talked about on Slack, `telnet` doesn't always work for this. Seems to depend on OS how well / if it works at all.

This has to do with the line endings. Linkam uses only a CR as a line terminator, while telnet will tend to use CRLF.

I think this is fine for this PR, but we should open an issue against this. The user needs a way to talk to the device on its primary protocol. 

We'll probably have to create our own little TCP Stream client that is aware of what line terminators the device its talking to expects.